<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>764</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #252832;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="sidebar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>764</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #181D29;</string>
    </property>
    <widget class="QWidget" name="sidebar_content" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>191</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="sidebar_logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Downloads/1DAF91D6-CDA2-4EF4-8A2E-AA1482B02B86 3 1.png);
background-repeat: no-repeat;
background-size: cover;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="sidebar_name">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>12</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px; 
font-weight: 100;
color: #fff;</string>
      </property>
      <property name="text">
       <string>тимур</string>
      </property>
     </widget>
     <widget class="QLabel" name="sidebar_surname">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>116</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>99</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
font-weight: 800;
color: #fff;</string>
      </property>
      <property name="text">
       <string>КОВАЛЕВ</string>
      </property>
     </widget>
     <widget class="QWidget" name="nav" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>191</width>
        <height>271</height>
       </rect>
      </property>
      <widget class="QPushButton" name="all_films">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>20</y>
         <width>171</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 24px; 
color: #fff;
border: none;</string>
       </property>
       <property name="text">
        <string>Все фильмы</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>-4</y>
      <width>1021</width>
      <height>775</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
	border: none;
}

QScrollBar:vertical {
    border: none;
    background: transparent;
    width: 20px;
    margin: 15px 0 15px 0;
    border-radius: 0px;
	margin-right: 10px;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(80, 80, 122);
	min-height: 30px;
	border-radius: 5px;
	transition: 0.3s;
	cursor: pointer;
}
QScrollBar::handle:vertical:hover{	
	transition: 0.3s;
	background-color: rgb(54, 57, 90);
}

QScrollBar::sub-line:vertical {
	opacity: 0;
	visibility: hidden;
	display: none;
}

QScrollBar::add-line:vertical {
	opacity: 0;
	visibility: hidden;
	display: none;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scroll_content">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1021</width>
       <height>1384</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="main">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="block" native="true">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>269</y>
           <width>925</width>
           <height>1130</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <widget class="QWidget" name="block_content" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>911</width>
            <height>1100</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #0F1D2D;
border-radius: 20px;</string>
          </property>
          <widget class="QLabel" name="block_img">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>191</width>
             <height>281</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/newPrefix/Downloads/image 2.png);
background-repeat: no-repeat;
background-size: cover;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QWidget" name="block_text_info" native="true">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>30</y>
             <width>591</width>
             <height>291</height>
            </rect>
           </property>
           <widget class="QLabel" name="title">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>501</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #fff;
font-size: 40px;</string>
            </property>
            <property name="text">
             <string>Богемская рапсодия (2018)</string>
            </property>
           </widget>
           <widget class="QLabel" name="year_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>80</y>
              <width>121</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 20px; color: #fff;</string>
            </property>
            <property name="text">
             <string>Год выпуска:</string>
            </property>
           </widget>
           <widget class="QLabel" name="country_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>115</y>
              <width>81</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 20px; color: #fff;</string>
            </property>
            <property name="text">
             <string>Страна:</string>
            </property>
           </widget>
           <widget class="QLabel" name="genre_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>150</y>
              <width>61</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 20px; color: #fff;</string>
            </property>
            <property name="text">
             <string>Жанр:</string>
            </property>
           </widget>
           <widget class="QLabel" name="director_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>185</y>
              <width>101</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 20px; color: #fff;</string>
            </property>
            <property name="text">
             <string>Режиссёр:</string>
            </property>
           </widget>
           <widget class="QLabel" name="budget_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>220</y>
              <width>81</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 20px; color: #fff;</string>
            </property>
            <property name="text">
             <string>Бюджет:</string>
            </property>
           </widget>
           <widget class="QLabel" name="year">
            <property name="geometry">
             <rect>
              <x>148</x>
              <y>83</y>
              <width>60</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255, 255, 255, 153);
font-size: 18px;</string>
            </property>
            <property name="text">
             <string>2018</string>
            </property>
           </widget>
           <widget class="QLabel" name="country">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>116</y>
              <width>361</width>
              <height>21</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255, 255, 255, 153);
font-size: 18px;</string>
            </property>
            <property name="text">
             <string>Великобритания, США</string>
            </property>
           </widget>
           <widget class="QLabel" name="genre">
            <property name="geometry">
             <rect>
              <x>87</x>
              <y>151</y>
              <width>361</width>
              <height>21</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255, 255, 255, 153);
font-size: 18px;</string>
            </property>
            <property name="text">
             <string>биография, музыка, драма</string>
            </property>
           </widget>
           <widget class="QLabel" name="director">
            <property name="geometry">
             <rect>
              <x>127</x>
              <y>186</y>
              <width>361</width>
              <height>21</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255, 255, 255, 153);
font-size: 18px;</string>
            </property>
            <property name="text">
             <string>Брайан Сингер</string>
            </property>
           </widget>
           <widget class="QLabel" name="budget">
            <property name="geometry">
             <rect>
              <x>108</x>
              <y>221</y>
              <width>361</width>
              <height>21</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255, 255, 255, 153);
font-size: 18px;</string>
            </property>
            <property name="text">
             <string>$52 000 000</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="video" native="true">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>350</y>
             <width>851</width>
             <height>481</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(167, 196, 255);</string>
           </property>
          </widget>
          <widget class="QLabel" name="description">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>830</y>
             <width>851</width>
             <height>231</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #fff;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>Чествование группы Queen, их музыки и их выдающегося вокалиста Фредди Меркьюри, который бросил вызов стереотипам и победил условности, чтобы стать одним из самых любимых артистов на планете. Фильм прослеживает головокружительный путь группы к успеху благодаря их культовым песням и революционному звуку, практически распад коллектива, поскольку образ жизни Меркьюри выходит из-под контроля, и их триумфальное воссоединение накануне концерта Live Aid, ставшим одним из величайших выступлений в истории рок-музыки.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="header" native="true">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>35</y>
           <width>925</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <widget class="QLabel" name="header_logo">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>351</width>
            <height>190</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(./images/logo.png);
background-repeat: no-repeat;
background-size: cover;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="header_search">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>770</width>
            <height>60</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff; 
border-radius: 18px;
padding: 0 20px;
padding-bottom: 5px;
font-size: 28px;
color: #000;</string>
          </property>
          <property name="placeholderText">
           <string>поиск...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="header_button">
          <property name="geometry">
           <rect>
            <x>790</x>
            <y>130</y>
            <width>131</width>
            <height>60</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #224369;
color: #fff;
font-size: 28px;
border: none;
border-radius: 15px;
padding-bottom: 2px;</string>
          </property>
          <property name="text">
           <string>найти</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../userimage.qrc"/>
 </resources>
 <connections/>
</ui>
